### ISSUE TITLE
Another idea from dnethack - using tinning kits to produce blood potions

### ISSUE THREAD

**Issue by cosmodreamer** [elunna/hackem#536]
Could be very nice for vamps. I admire dnethack's blood potion production and stocking when playing as a vamp. Would be great to see it in Hack'em - if it would be consistent with the Dev vision of course.

**Comment by elunna** [comment]
I don't really maintain this repo anymore - I've moved onto NerfHack. However, malor forked HackEM and does maintain it here: https://github.com/nethack-cleaner/HackEM.

I think malor did in fact implement dnh style tinning for vampires.

**Comment by transcendreamer** [comment]
Yes, I see dnh style tinning is really implemented there! Thanks!
